routes/transaction.go

```diff
@@ -3171,6 +3171,22 @@ type AssociationLimitMapItem struct {
 	OpCount                 uint64
 }
 
+type AccessGroupLimitMapItem struct {
+	AccessGroupOwnerPublicKeyBase58Check string
+	ScopeType                            lib.AccessGroupScopeString
+	AccessGroupKeyName                   lib.GroupKeyName
```

Comment:
Let's use strings when passing to the frontend. We convert from lib.GroupKeyName to a string like so `string(lib.AccessKeyNameDecode(groupKeyName))`